docs/admin/cluster-management.md

```diff
@@ -14,47 +14,141 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Management
 
-This doc is in progress.
+## Creating and configuring a Cluster
+
+Assuming that you have machines for your clusters ready, you need to install Kubernetes on them. For specific instructions you can consult one of existing [Getting Started guides](../../docs/getting-started-guides/README.md) for various environments:
```

Comment:
I would remove everything in this subsection after this sentence. It will be annoying to try to keep this list up-to-date, and the From Scratch doc is already pointed to from docs/getting-started-guides/README.md